### PR TITLE
Support

### PR DESCRIPTION
**Pull request by moritzbae**

### CODE REVIEW

File: config/data/markdown/en/about.md [hunk 1]
```diff
@@ -53,6 +53,15 @@ I currently work at [anders und sehr GmbH](https://andersundsehr.de) as my main
 I do not work fulltime as a freelancer, and I do not plan to change that in the foreseeable future.
 
 ~Timeline~
+---
```
Comment (ChrisB9) [inline config/data/markdown/en/about.md:56]:
```suggestion
```

**Review by ChrisB9 — CHANGES_REQUESTED** [review]
Thanks a lot!

File: config/data/markdown/en/about.md (hunk 1 above)
Comment (moritzbae) [reply config/data/markdown/en/about.md:56]: To keep it consistent the same change should be done on the german about.md.
So you're sure to want it like this?

Comment (ChrisB9) [reply config/data/markdown/en/about.md:56]: Yup, it's necessary to prevent an empty timeline slot

**Review by ChrisB9 — APPROVED** [review]

**Pull request merged**